app/src/main/res/values/colors_palette.xml

```diff
@@ -31,8 +31,8 @@
     <color name="yellow_alpha_15">#26ffeb3b</color>
 
     <color name="purple_alpha_30">#2658037C</color>
-
     <color name="purple_alpha_50">#8058037C</color>
+    <color name="dark_purple_alpha_0">#006c78b6</color>
```

Comment:
No🙅
If you make this transparent, the gradation will not work.

`@android:color/transparent` | `@color/dark_purple_alpha_0`
:--: | :--:
<img src="https://cloud.githubusercontent.com/assets/11440952/22944204/4822a644-f333-11e6-9b32-3bb9a7169d6a.png" width="300" /> | <img src="https://cloud.githubusercontent.com/assets/11440952/22944205/4828e478-f333-11e6-9c8b-a55b5150b89f.png" width="300" />